### PR TITLE
Prep to publish

### PR DESCRIPTION
**Pull request by grouma**
cc @jonahwilliams 

Towards https://github.com/flutter/flutter/issues/77856

### CODE REVIEW

File: dwds/pubspec.yaml [hunk 1]
```diff
@@ -31,26 +31,26 @@ dependencies:
   shelf_packages_handler: '>=1.0.0 <3.0.0'
   shelf_proxy: '>=0.1.0+6 < 2.0.0'
   shelf_static: ^0.2.8
```
Comment (jonahwilliams) [inline dwds/pubspec.yaml:33]: What about shelf_static?

Comment (grouma) [reply dwds/pubspec.yaml:33]: Note I can't depend on `shelf_static` version `^1.0.0` due to `devtools_server`.

**Review by jakemac53 — APPROVED** [review]

File: dwds/pubspec.yaml [hunk 2]
```diff
@@ -28,29 +28,29 @@ dependencies:
   pool: ^1.4.0
   pub_semver: ^2.0.0
   shelf: '>=0.7.0 < 2.0.0'
```
Comment (kevmoo) [inline dwds/pubspec.yaml:30]: Might as well bump this, too. `^1.0.0`

**Comment by jonahwilliams** [comment]
devtools has been published FYI

**Pull request merged**